v1/ao/internal/config/config.go

```diff
@@ -36,6 +36,8 @@ const (
 	maxConfigFileSize = 1024 * 1024
 	// the default collector url
 	defaultSSLCollector = "collector.appoptics.com:443"
+	maxTokenBucketCapacity = 8
```

Comment:
Can the logic in https://github.com/appoptics/appoptics-apm-go/blob/95a4612ca5d00f24b279d23a371cdac05e665036/v1/ao/internal/config/validators.go#L114 uses this too? So it wouldn't have 2 values? :)

Also wondering if someone sets to a negative value, should we use the max instead? It makes sense to set max if someone tries to set to an even greater value , but the other way around might be a bit unexpected. Not a big deal if it's hard to change that 😅

Reply:
> Can the logic in ... uses this too?

Good point. I'll make a change.

> Also wondering if someone sets to a negative value, should we use the max instead?

My original idea (and implementation before this change) was that if the config value is not expected (either in a wrong data type or out of range) the default value will be used, as setting it to either the upper limit value or other values may not be the intent of the user. It sounds like a different flavour only and I'm fine with either one.